bin/install

```diff
@@ -28,18 +39,24 @@ fetch_patches() {
       >&2 echo "Using patch from URL: $line"
       curl -fSs "$line" || exit 1
     else
-      local abs_path=$(get_absolute_path "$line")
+      local abs_path
+      abs_path="$(get_absolute_path "$line")"
       >&2 echo "Using local patch: $abs_path"
       cat "$abs_path" || exit 1
     fi
-  done <<< "${RUBY_APPLY_PATCHES:-}"
+  done <<< "$@"
 }
 
 get_absolute_path() {
-  local start_dir=$(pwd)
```

Comment:
I've break this into multiple declaration due to [SC2155](https://github.com/koalaman/shellcheck/wiki/SC2155)